### DISCUSSION TITLE
posts/2024-08-13-crowds-vs-friends/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#17] [Announcements]
# posts/2024-08-13-crowds-vs-friends/

Our immune systems evolved for a different world (that didn’t involve 100,000 global flights per day).

https://rachel.fast.ai/posts/2024-08-13-crowds-vs-friends/

**Comment by wiseyoungbuck** [comment]
Very informative take thanks.

**Reply by racheltho** [reply]
Thank you!

**Comment by Muero9** [comment]
I think I found a typo. I believe “in term” should be “in turn”. 

But thanks for teaching me stuff!

**Reply by racheltho** [reply]
Thanks!  I have updated the typo.

**Comment by Rhyothemis** [comment]
Great post!
I've often wondered about how our relative lack of exposure to ectoparasites (bedbugs, lice, etc.) in moderns times might affect overall immune function.

**Comment by tinawind** [comment]
Thank you so much for this!   I've shared the link and snippets of the text (the whole thing is too long for a facebook commnet) dozens of times at least, in response to the current popular believe that you have to keep getting sick in order to be healthy.

**Reply by racheltho** [reply]
Thank you for sharing the post.  It is frustrating how widespread that harmful belief has become.
